ansible/roles/edu-sharing/templates/mirror-vocabs.sh.j2

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
 {% for mirror in mirror_vocabularies %}
-wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
-chown -R www-data:www-data {{ mirror.target_directory }}
+    wget -q --mirror "{{ mirror.url }}" -P {{ mirror.target_directory }}
+    chown -R www-data:www-data {{ mirror.target_directory }}
+    {% if ((mirror.isSorted | default("false") == 'true') and (mirror.variant | default("false") == 'true')) %}
+        path={{ mirror.url | regex_replace('^(.*?)https://(.*)$', '\\1\\2') | regex_replace('^(.*?)scheme.json(.*)$', '\\1\\2')}}
```

Comment:
two things:

1. in the regex, why do you use the first group? `(.*?)` this will always ever be empty, because there will be no content before the protocol in the url. Or do I miss something? Please remove the first group -> this will reduce complexity
2. To get the directory of a file in ansible, please use the ansible-function `dirname` instead of this regex. So, `| dirname ` instead of `| regex_replace('^(.*?)scheme.json(.*)$'` => this is much better/easier to read and less complex